integration_tests/dbt_project.yml

```diff
@@ -1,17 +1,17 @@
 name: 'hubspot_integration_tests'
-version: '0.17.0'
+version: '0.17.1'
 
 profile: 'integration_tests'
 config-version: 2
 vars:
-  hubspot_schema: hubspot_integration_tests_11
+  hubspot_schema: hubspot_integration_tests_55
   hubspot_source:
     hubspot_service_enabled: true
-    hubspot_sales_enabled: true # enable when generating docs
-    hubspot_marketing_enabled: true # enable when generating docs
-    hubspot_contact_merge_audit_enabled: true # enable when generating docs
-    hubspot_using_all_email_events: true # enable when generating docs
-    hubspot_merged_deal_enabled: true # enable when generating docs  
+    # hubspot_sales_enabled: true # enable when generating docs
+    # hubspot_marketing_enabled: true # enable when generating docs
+    # hubspot_contact_merge_audit_enabled: true # enable when generating docs
+    # hubspot_using_all_email_events: true # enable when generating docs
+    # hubspot_merged_deal_enabled: true # enable when generating docs  
```

Comment:
These should be commented back in to properly test during integration tests

Reply:
@fivetran-joemarkiewicz  I just had these enabled to generate docs, but commented out otherwise since I assumed they weren't needed for tests as they're explicitly configured in the [run script](https://github.com/fivetran/dbt_hubspot/blob/bugfix/integer_overflow/.buildkite/scripts/run_models.sh)?

Reply:
Got it, I was wondering why they were not commented out previously. This looks good then, thanks!